### PR TITLE
Goal of this PR is to make sure we raise the onNewXXX when the entity…

### PR DESCRIPTION
**Pull request by deltakosh**
… is fully constructed

### CODE REVIEW

**Comment by deltakosh** [comment]
Discussion: https://forum.babylonjs.com/t/gpu-picker-possible-bug/54814/4

File: packages/dev/core/src/scene.ts [hunk 1]
```diff
@@ -2650,7 +2651,9 @@ export class Scene implements IAnimatable, IClipPlanesHolder, IAssetContainer {
             newMesh._addToSceneRootNodes();
         }
 
-        this.onNewMeshAddedObservable.notifyObservers(newMesh);
+        DelayCall(() => {
```
Comment (ryantrem) [inline packages/dev/core/src/scene.ts:2654]: Is it the next browser event loop, or the next BJS frame we need to wait for? Does waiting for the next browser event loop make assumptions about how the render loop is working? Should this use onAfterRenderObservable.addOnce?

Comment (deltakosh) [reply packages/dev/core/src/scene.ts:2654]: We only need to get of the current calling stack (the constructor)

**Comment by deltakosh** [comment]
OMG you are right! Thanks for the double check

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15848/merge/testResults/webgpuplaywright/index.html

**Review by sebavan — APPROVED** [review]

**Pull request merged**